client/android/app/src/main/java/com/snapscreen/aerogear/AerogearModule.java

```diff
@@ -0,0 +1,56 @@
+package com.snapscreen.aerogear;
+
+import android.util.Log;
+
+import com.facebook.react.bridge.ReactApplicationContext;
+import com.facebook.react.bridge.ReactContextBaseJavaModule;
+import com.facebook.react.bridge.ReactMethod;
+import com.facebook.react.bridge.ReadableMap;
+
+import com.facebook.react.bridge.Callback;
+import org.jboss.aerogear.android.unifiedpush.PushRegistrar;
+import org.jboss.aerogear.android.unifiedpush.RegistrarManager;
+import org.jboss.aerogear.android.unifiedpush.fcm.AeroGearFCMPushConfiguration;
+
+import java.net.URI;
+
+public class AerogearModule extends ReactContextBaseJavaModule {
+
+    private final String VARIANT_ID       = "3e2c6bd7-cd69-49eb-bd9e-a9360e79f1a9";
+    private final String SECRET           = "15448edc-8896-4163-aca6-cbf783f9165b";
+    private final String GCM_SENDER_ID    = "842964426922";
+    private final String UNIFIED_PUSH_URL = "http://ups-server-snapscreen.74.207.224.48.xip.io/ag-push/";
+
+    public AerogearModule(ReactApplicationContext reactContext) {
+        super(reactContext);
+    }
+
+
+    @Override
+    public String getName() {
+        return "Aerogear";
+    }
+
+    @ReactMethod
+    public void init(ReadableMap config, Callback successCallback, Callback cancelCallback) {
+        RegistrarManager.config("register", AeroGearFCMPushConfiguration.class)
```

Comment:
Another note, we should pass in on the an alias config so we can set the alias here.